great-idea/services.html

```diff
@@ -16,91 +16,116 @@
 
 <body>
```

Comment:
Recommend having a `main` tag thet encompasses everything inside the body:

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/main